src/theory/fp/theory_fp_rewriter.cpp

```diff
@@ -175,8 +175,9 @@ namespace rewrite {
     if (node[0] == node[1]) {
       return RewriteResponse(REWRITE_DONE, NodeManager::currentNM()->mkConst(true));
     } else if (!isPreRewrite && (node[0] > node[1])) {
-	Node normal = NodeManager::currentNM()->mkNode(kind::EQUAL,node[1],node[0]);
-	return RewriteResponse(REWRITE_DONE, normal);
+      Node normal =
+          NodeManager::currentNM()->mkNode(kind::EQUAL, node[1], node[0]);
+      return RewriteResponse(REWRITE_AGAIN_FULL, normal);
```

Comment:
This change should not be needed (there is no other rewrite that could apply and `node[0]`/`node[1]` are already rewritten at that point.

Reply:
Yes, you're right.